main.go

```diff
@@ -30,14 +30,12 @@ func main() {
 		sig := <-sc
 		fmt.Printf("\nGot signal [%v] to exit.\n", sig)
 		log.Warn("received signal to exit", zap.Stringer("signal", sig))
-		switch sig {
-		case syscall.SIGTERM:
-			cancel()
-			os.Exit(0)
-		default:
-			cancel()
-			os.Exit(1)
-		}
+		cancel()
+		fmt.Println("gracefully shuting down, press ^C again to force exit")
```

Comment:
print to stderr please.

```suggestion
		fmt.Fprintln(os.Stderr, "gracefully shuting down, press ^C again to force exit")
```